### ISSUE TITLE
Package cppcheck-htmlreport

### ISSUE THREAD

**Issue by timsnyder** [conda-forge/cppcheck-feedstock#20]
Issue:

We use the [cppcheck-htmlreport](https://github.com/danmar/cppcheck/tree/master/htmlreport) script.  It would be great if the conda package included it (or made it available as a separate output package cppcheck-htmlreport).

It might also be really nice for the conda package to include a CppcheckConfig.cmake or cppcheck-config.cmake file in the right place so that cmake will find it without having to provide FindCppcheck.cmake in every project..

I'll submit a PR for this.

**Comment by tovrstra** [comment]
See #21. For the htmlreport script. Sorry, I did not notice your last sentence till now.

Feel free to make a PR for the `CppcheckConfig.cmake` or `cppcheck-config.cmake` file. I'm not sure where these should go.

**Comment by timsnyder** [comment]
Cool.  Thanks for adding the htmlreport script @tovrstra .  I'll figure out the `CppcheckConfig.cmake` and post a PR soon.

**Comment by tovrstra** [comment]
#21 just got merged. Can you try if the htmlreport script works as you would expect in the updated package? (These will roll out in a few days I guess.)

Any ideas on how to deal with `CppcheckConfig.cmake`? Thanks.

**Comment by timsnyder-siv** [comment]
Hi @tovrstra.  Sorry for the delay in replying.  A coworker just told me that he's trying the updated package and still doesn't see the cppcheck-htmlreport script. 

I looked in a couple of system package recipes (debian and fedora) and they are both just copying the script.  Looking at the setup.py, it's missing the stuff to tell it to install the script and I don't know that it is worth trying to upstream a change for something this small.  I think if you change `python setup.py install` to `  install -D -p -m 755 cppcheck-htmlreport $PREFIX/bin/cppcheck-htmlreport` then it will work.

Regarding the `CppcheckConfig.cmake`, it's been long enough since I looked at a cmake build that I've forgotten.  Let's not worry about that part.
